lightningd/closing_control.c

```diff
@@ -596,6 +624,10 @@ static struct command_result *json_close(struct command *cmd,
 				    channel->lease_expiry,
 				    get_block_height(cmd->ld->topology));
 
+	assert(channel->final_key_idx <= UINT32_MAX);
+	index_val = (u32) channel->final_key_idx;
```

Comment:
maybe comment somewhere that this value can be overwritten below and under what conditions

Reply:
done